pkg/sync/syncWorker.go

```diff
@@ -257,7 +257,12 @@ func (s *syncWorker) setupNodeRegistration(
 				// this indicates that the node is shutting down
 				// the notifierCtx should have been shut down already,but it can't hurt to cancel it just in case
 				notifierCancel()
-			case <-registryChan:
+			case _, ok := <-registryChan:
```

Comment:
what does this case mean? We lost connection to the registry? does that mean that we need to rebuild the node connection? do we have to call `notifierCancel`?